src/QMCHamiltonians/tests/test_ion_derivs.cpp

```diff
@@ -772,4 +772,778 @@ TEST_CASE("Eloc_Derivatives:multislater_wj", "[hamiltonian]")
 #endif */
 }
 
+TEST_CASE("Eloc_Derivatives:proto_sd_noj", "[hamiltonian]")
```

Comment:
This is one `:` from being a unit test of class Eloc_Deriatives.  There is no such class and this is an integration test of the derivative calculation of various hamiltonians and TWF.  Update naming to make that clear.  Update PR name to reflect as well.

Reply:
Happy to make these changes.  But because I'm dumb: what sort of naming convention would work for this?  Admittedly, I named the tests like this because I glommed onto the existing convention, but now's a good time to update all the names in this file.

Reply:
Well we are only now admitting a difference between integration and unit tests.  Filenames so far have followed the old snake case convertion. Which doesn't make it look like a test of a specific Type but which also makes it look like a legacy test file.

My example is:
`test_dual_allocators_ohmms_containers.cpp`

This was an attempt to name a test case.
`TEST_CASE("OhmmsMatrix_VectorSoaContainer_View", "[Integration][Allocators]")`

so something like
Type1_Type2_Operation

If you can propose something smarter that would be great.  Certainly the file name convention doesn't accomplish anything.